specification/2.0/schema/image.schema.json

```diff
@@ -29,5 +28,8 @@
     "additionalProperties": false,
     "dependencies": {
         "bufferView": [ "mimeType" ]
+    },
+    "not": {
```

Comment:
Current approach only disallows presence of both fields at the same time.
"anyOf" with two "required" entries has different meaning (either is required) and doesn't prevent ambiguous case. 
```json
{
  "anyOf": [
    {"required":["uri"]},
    {"required":["bufferView"]}
  ]
}
```